common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -56,33 +56,32 @@ export const BaseButton = styled.button.attrs<BaseButtonProps>(props => ({
 `;
 
 type BaseButtonInnerProps = {
-  isInline?: boolean;
-  isPill?: boolean;
+  $isInline?: boolean;
+  $isPill?: boolean;
 };
 
 const BaseButtonInnerSpan = styled.span<BaseButtonInnerProps>``;
 export const BaseButtonInner = styled(
   BaseButtonInnerSpan
 ).attrs<BaseButtonInnerProps>(props => ({
-  className: font(props.isInline ? 'intr' : 'intb', props.isPill ? 6 : 5),
+  className: font(props.$isInline ? 'intr' : 'intb', props.$isPill ? 6 : 5),
 }))`
   display: flex;
   align-items: center;
   height: 1em;
 `;
 
 type ButtonIconWrapperAttrsProps = {
-  isTextHidden?: boolean;
-  iconAfter?: boolean;
-  isPill?: boolean;
+  $isTextHidden?: boolean;
+  $iconAfter?: boolean;
```

Comment:
As I went through this I noticed a few props were never used, and I removed them when I felt it was the right move, but do let me know if rendering them on the DOM is necessary (like maybe it's used as a selector, or for analytics?)